src/pattern-library/util/jsx-to-string.tsx

```diff
@@ -46,14 +48,26 @@ function isJSXElement(value: any): value is VNode<any> {
   return typeof elementType === 'string' || typeof elementType === 'function';
 }
 
+/**
+ * Should the prop with `name` be ignored from output code strings?
+ *
+ * The special `children` prop, as well as any prop name beginning with an
+ * underscore, should be ignored.
+ */
+function isExcludedProp(name: string) {
+  return name.match(/^_/) || name === 'children';
```

Comment:
Maybe `name.startsWith('_')` is a bit more clear than the RegExp?

I personally would understand this particular RegExp, so I leave it up to you.